tests/setup_nodes.py

```diff
@@ -496,6 +501,38 @@ async def setup_farmer_harvester(consensus_constants: ConsensusConstants, start_
     await _teardown_nodes(node_iters)
 
 
+async def setup_farmer_multi_harvester(harvester_count: int, temp_dir: Path, consensus_constants: ConsensusConstants):
```

Comment:
Or is there something more particular than `Service` since we know what they are going to be?

```suggestion
async def setup_farmer_multi_harvester(
    harvester_count: int,
     temp_dir: Path,
     consensus_constants: ConsensusConstants,
) -> Iterator[Tuple[List[Service], Service]]:
```